saw-remote-api/python/saw/__init__.py

```diff
@@ -359,6 +380,43 @@ def llvm_load_module(bitcode_file: str) -> LLVMModule:
     return LLVMModule(bitcode_file, name)
 
 
+def llvm_assume(module: LLVMModule,
+                function: str,
+                contract: llvm.Contract,
+                lemma_name_hint: Optional[str] = None) -> VerificationResult:
+    if lemma_name_hint is None:
+        lemma_name_hint = contract.__class__.__name__ + "_" + function
+    name = __fresh_server_name(lemma_name_hint)
+
+    result: VerificationResult
+    conn = __get_designated_connection()
+    conn_snapshot = conn.snapshot()
```

Comment:
This usage of `snapshot` should probably go, since we don't actually support connection snapshots at the moment.

See changes here inside `llvm_verify`'s implementation: https://github.com/GaloisInc/saw-script/pull/1174

Reply:
And when we address [this issue](https://github.com/GaloisInc/saw-script/issues/1192) we should probably tweak the `llvm_assume` code as well, but that can be done in a subsequent PR obviously.

Reply:
Ah, I wondered about that when I was adapting the code. I'll remove it.